HW2/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/students
+spring.datasource.username=postgres
+spring.datasource.password=1
```

Comment:
Do not save plain text passwords (or credentials at all) in repo, even for local environment.

Prefer to use environment variables instead:
`password=${DB_PASSWORD}`

- It will be safe for posting in the repo (even in public one)
- It is a cloud-ready approach (12 factors app)
- It will be easy to setup it for local run from IDE or in condole